.circleci/config.yml

```diff
@@ -171,17 +183,29 @@ references:
             echo "$(date +%s)-$(cat .version)-dev-$(git rev-parse --short HEAD)" > .version
             cat .version
             ytool -s appVersion "$(cat .version)" -s version "$(cat .version)" -f "./codacy/Chart.yaml" -e
+      - run:
+          name: Set environment specific helm values
+          command: |
+            # DELETE ME: this is a temporary fix because breaking changes were added to ci-do so we can't add gettext
```

Comment:
Delete what exactly, the entire command block? The entire run block?

Reply:
As above, just delete the installation of the package. I can't add a package to ci-do without losing helm2, which I need.